scripts/st2bootstrap-deb.sh

```diff
@@ -605,6 +611,11 @@ install_st2() {
   sudo crudini --set /etc/st2/st2.conf database username "stackstorm"
   sudo crudini --set /etc/st2/st2.conf database password "${ST2_MONGODB_PASSWORD}"
 
+  # Configure [messaging] section in st2.conf (username password for RabbitMQ access)
+  RABBITMQHOST="${RABBITMQHOST:-rabbitmq}"
+  AMQP="amqp://stanley:$ST2_RABBITMQ_PASSWORD@$RABBITMQHOST:5672/"
+  sudo sed -i "/\[messaging\]/,/\[.*\]\|url/ {n; s#url.*=.*#url = $AMQP#}" /etc/st2/st2.conf
```

Comment:
Can you use the crudini command to set the messaging URL in the st2.conf file?  You can see similar example for setting the mongodb setting.